docs/getting_started.md

```diff
@@ -42,7 +42,7 @@ you're working on.
 pip install streamlit
 ```
 
-Now run the hello world app just to make sure everything is working:
```

Comment:
Hm, should we keep this kind of usage?

I think removing "just" makes sense when the implication of the word is that "this should be easy for you to do!" e.g. "Just do X". I absolutely agree we should remove these from our docs.

But another use of just, as in here, is a colloquial way of saying "only" -- we're trying to communicate that the purpose of the `hello world` is only to verify that the installation worked. "Just" here somewhat softens what would otherwise be a straight command, so I'd lean towards keeping it.

Reply:
To me, it doesn't feel like it adds anything. The statement is already forceful to "run this app"...if anything, I would go further to say it's worth running the app to learn about Streamlit functionality and to learn idioms for working. 

So not "just" to make sure things are set up, but for much more. But for this PR, I was only removing words I felt don't add clarity to the docs.

Reply:
Okay, makes sense!